azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeAudioSelectionChangedEvent.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * Event with audio selection changed.
+ */
+public final class CallCompositeAudioSelectionChangedEvent {
```

Comment:
Audio is a bit vague, given there is both Audio Inputs (Microphones) and Audio Outputs (Speakers). But I do acknowledge these are often bundled in the context of mobile (I.e. we aren't saying USB Mic + BT Speakers, not headset on mobile).

While I know we bundle, it might make sense to decouple Input/Output here in the API, as in Teams native there is robust input/output selections, and it might be something we want to replicate in the future.

I.e. 
CallCompositeAudioSelectionChangedEvent {
  InputDevice/OutputDevice
}

Even if that means that they are == for now in the native SDK's, it's clear where audio is coming and going to in both directions, and leaves headroom for more advanced audio routing in the future.

Reply:
I think we had discussion before Adam, SDK as well as WEB use audio as default for output. Whenever input is required mic is indicated. Will book a meeting to discuss with Pavel and finalize. 

Native SDK just used
```
setTelecomManagerAudioRoute
```

Android OS
```
CallAudioState
```

If required we can add output may be? but seems like Audio is by default indicated as output.

Reply:
I see a point in making it more clear with input/output, but on the other hand we can just follow existing name convention Android is using.